dags/bqetl_addons.py

```diff
@@ -101,7 +101,11 @@
         dataset_id="telemetry_derived",
         project_id="moz-fx-data-shared-prod",
         owner="jklukas@mozilla.com",
```

Comment:
One more Jeff is left here

Reply:
💩  oops. Should be fixed now. Thanks for being so thorough 🙇